domains/l6174.json

```diff
@@ -3,9 +3,11 @@
   "repo": "https://github.com/l6174/l6174.github.io",
   "owner": {
     "username": "l6174",
-    "email": "l.lawliet6174@gmail.com"
+    "email": "l6174@proton.me"
   },
   "record": {
-    "CNAME": "l6174.github.io"
+    "CNAME": "l6174.github.io",
```

Comment:
You cannot use a CNAME record with other records.

Reply:
I am left with no choice but to remove MX and SPF records. Is there any other alternative BTW?

Reply:
> I am left with no choice but to remove MX and SPF records. Is there any other alternative BTW?

You know you can also use A records for Github Pages, right?

Reply:
> > I am left with no choice but to remove MX and SPF records. Is there any other alternative BTW?
> 
> You know you can also use A records for Github Pages, right?

Isn't that only for Apex domains?

Reply:
It works also on subdomains (a lot of users have been doing this)

Reply:
Ok I will try that and open another PR. Thanks!